lib/sharepoint/client.rb

```diff
@@ -351,10 +373,8 @@ def upload(filename, content, path, site_path = nil)
       path = path[1..-1] if path[0].eql?('/')
       url = uri_escape "#{url}GetFolderByServerRelativeUrl('#{path}')/Files/Add(url='#{sanitized_filename}',overwrite=true)"
       easy = ethon_easy_json_requester
-      easy.headers = {
-        'accept' => 'application/json;odata=verbose',
-        'X-RequestDigest' => xrequest_digest(site_path)
-      }
+      easy.headers = with_bearer_authentication_header({ 'accept' => 'application/json;odata=verbose',
+                                                         'X-RequestDigest' => xrequest_digest(site_path) })
```

Comment:
in order to reduce the number of lines changed, can we have the same indentation as above?